spec/services/course/statistics/assessment_score_summary_download_service_spec.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe Course::Statistics::AssessmentsScoreSummaryDownloadService do
+  let(:instance) { Instance.default }
+  with_tenant(:instance) do
+    let!(:course) { create(:course) }
+    let!(:course2) { create(:course) }
+    let!(:student1) { create(:course_user, course: course, name: 'Student 1') }
+    let!(:student2) { create(:course_user, :phantom, course: course, name: 'Student 2') }
+    let!(:student3) { create(:course_user, course: course, name: 'Student 3') }
+    let!(:student4) { create(:course_user, course: course2, name: 'Student 3') }
```

Comment:
Can remove `student4`, not important for this test. In fact, student4's user is not the same as student3's user.

Reply:
the purpose of `student4` there is to ensure that, within the process, only the student inside the course will get included (by meaning that no student from other course will get included inside the table)

Reply:
Then why is the name Student 3?

Reply:
this is to test that even if the student's name is the same across the courses, the downloading job will still fetch the correct course that we demanded. If the name is different (let's say, Student 4), then it won't test anything.

Nevertheless, I will change the name into `student32` to still ensure that it's 'Student 3'